container/docker/handler.go

```diff
@@ -160,8 +163,12 @@ func newDockerContainerHandler(
 		fsInfo:             fsInfo,
 		rootFs:             rootFs,
 		rootfsStorageDir:   rootfsStorageDir,
-		fsHandler:          newFsHandler(time.Minute, rootfsStorageDir, otherStorageDir, fsInfo),
 		envs:               make(map[string]string),
+		ignoreMetrics:      ignoreMetrics,
+	}
+
+	if _, ignore := ignoreMetrics[container.DiskUsageMetrics]; !ignore {
```

Comment:
`bool` is misleading since the zero value for an unknown key is also bool. I'm adding a complex type to make this cleaner.